app/pages/login/login.js

```diff
@@ -78,19 +82,36 @@ export let Login = translate()(class extends React.Component {
       <div className="login-simpleform">{form}</div>
     );
 
+    // for those accepting an invite, forward to keycloak login when available
+    if (
+      this.props.isInvite &&
+      keycloakConfig.initialized &&
+      !loggingIn &&
+      !this.props.isAuthenticated
+    ) {
+      keycloak.login({ loginHint: this.props.seedEmail });
+    }
+
+    // forward to keycloak login when available
+    if(keycloakConfig.initialized && !loggingIn && !this.props.isAuthenticated){
+      keycloak.login();
+      return (<></>);
```

Comment:
There's not a particular reason. The code at one point was returning other JSX and the solution collapsed on returning nothing, so I just made it an empty fragment. Functionally it renders the same as returning `null`, but technically creates an object to track rendering nothing, so I can switch it out to avoid confusion.